nixos/modules/services/computing/slurm/slurm.nix

```diff
@@ -226,6 +250,13 @@ in
 
     services.munge.enable = mkDefault true;
 
+    # use a static uid as default to ensure it is the same on all nodes
+    users.users.slurm = mkIf (cfg.user == defaultUser) {
+      name = defaultUser;
+      group = "nogroup";
```

Comment:
I think it would be better to use the dedicated `slurm` group. `nogroup` is shared by other users, which may result in unwanted access rights.

Reply:
Thanks for spotting that! It's now consistently set to group slurm.